code/game/objects/effects/decals/turfdecal/markings.dm

```diff
@@ -457,3 +457,9 @@
 
 /obj/effect/turf_decal/siding/wideplating_new/terracotta/end
 	icon_state = "siding_wideplating_new_end"
+
+/obj/effect/turf_decal/numbers
+	icon_state = "number_0-1"
+
+/obj/effect/turf_decal/numbers/two_nine
+	icon_state = "number_2-9"
```

Comment:
I don't like that these use dirs tbh, make them all separate objects.

Reply:
id rather not absolutely dump the dmi file with numbers besides its pretty fine to use in mapping

Reply:
too bad >:(